### PR TITLE
Update log-collection-troubleshooting-guide.md

### PR DESCRIPTION
**Pull request by devmf027**
Includes log collection application directory name matching the service value inside the yaml configuration file

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Comment by cswatt** [comment]
Thank you for opening this PR! A member of our team will review it shortly.

File: content/en/logs/guide/log-collection-troubleshooting-guide.md [hunk 1]
```diff
@@ -235,6 +235,19 @@ These are a few of the common configuration issues that are worth triple-checkin
...
+  conf.d/<APP_NAME>/conf.yaml
+  
+  conf.yaml file:
+   ```yaml
+   logs:
+     - type: file
+       path: "<PATH_LOG_FILE>/<LOG_FILE_NAME>.log"
+       service: "<APP_NAME>"
+       source: "<SOURCE>"
+
```
Comment (maycmlee) [inline content/en/logs/guide/log-collection-troubleshooting-guide.md:249]: Could you add closing backticks (```)?

Comment (maycmlee) [reply content/en/logs/guide/log-collection-troubleshooting-guide.md:246]: Thanks @devmf027! I committed a couple of editorial edits, but could you take a look at this one?

Comment (devmf027) [reply content/en/logs/guide/log-collection-troubleshooting-guide.md:247]: Backticks closed

File: content/en/logs/guide/log-collection-troubleshooting-guide.md [hunk 2]
```diff
@@ -235,6 +235,17 @@ These are a few of the common configuration issues that are worth triple-checkin
 
 4. You may have some .yaml parsing errors in your configuration files. YAML can be finicky, so when in doubt rely on a [YAML validator][8].
 
+5. The name of the directory in the `conf.d/` folder must match the service value in the `conf.yaml` file within that directory. For example, for `conf.d/<APP_NAME>/conf.yaml` the `conf.yaml` file has `service` set to `APP_NAME`:
```
Comment (maycmlee) [inline content/en/logs/guide/log-collection-troubleshooting-guide.md:238]:
```suggestion
5. The name of the directory in the `conf.d/` folder must match the service value in the `conf.yaml` file within that directory. For example, for `conf.d/<APP_NAME>/conf.yaml` the `conf.yaml` file has `service` set to `<APP_NAME>`:
```

**Pull request closed without merging**